wallet-api/src/Wallet/Emulator/Types.hs

```diff
@@ -309,21 +328,11 @@ blockchainActions = Op.singleton BlockchainActions
 assertion :: Assertion -> Trace ()
 assertion = Op.singleton . Assertion
 
--- | Make an assertion about the total value of funds owned by a wallet.
-assertOwnFunds :: Wallet -> (Value -> Maybe T.Text) -> Trace ()
-assertOwnFunds w p = assertion go where
-    go s = case Map.lookup w $ emWalletState s of
-                    Nothing -> Just $ AssertionError "Wallet not found"
-                    Just ws -> AssertionError <$> p (getSum $ foldMap Sum $ walletStateOwnAddresses ws)
-
--- | Assert that the funds owned by a wallet are equal to an amount
 assertOwnFundsEq :: Wallet -> Value -> Trace ()
-assertOwnFundsEq w target = assertOwnFunds w p where
-    tshow :: Show a => a -> T.Text
-    tshow = T.pack . show
-    p v
-        | v == target = Nothing
-        | otherwise   = Just $ T.unwords ["Funds in wallet", tshow w, "were", tshow v, ". Expected:", tshow target]
+assertOwnFundsEq wallet value = assertion $ OwnFundsEqual wallet value
```

Comment:
You might as well eta-reduce this like you did for the other one.

Reply:
done